packages/@aws-cdk/aws-appmesh/lib/virtual-node.ts

```diff
@@ -258,7 +215,7 @@ export class VirtualNode extends VirtualNodeBase {
     this.mesh = props.mesh;
 
     this.addBackends(...props.backends || []);
-    this.addListeners(...props.listener ? [props.listener] : []);
+    this.addListeners(...props.listener ? [props.listener] : [VirtualNodeListener.httpNodeListener()]);
```

Comment:
We should not default to a listener for virtual nodes. It is valid to create a virtual node without a listener. For example, we may have a service that polls SQS to do some work, and therefore would not need a listener